### ISSUE TITLE
VIP 5.0 and branches

### ISSUE THREAD

**Issue by jungshadow** [votinginfoproject/vip-specification#292]
First, [congrats on the official 5.0 release](http://bukk.it/bomp--bomp--bomp-bompbompbomp.gif)!

Second, noting that the main repository link goes to the old documentation, are you planning to merge vip5 to master? Or salt the earth and make a new master branch? Or...something else?

Again, excellent work!

**Comment by jdmgoogle** [comment]
If you've got a few free minutes I nominate you to do the GitHub-fu to dump the official 5.0 release into master. :)

**Comment by jungshadow** [comment]
@jdmgoogle Oh dear god. I tried a month back out of sheer curiosity, mostly to see how messed up the history was. Let's just say, [it didn't go well](http://bukk.it/noidea-commandant.jpg). Whilst my GitHub-fu is fairly strong, this adversary may be beyond my skill. This is why I suggested the "salt the earth" tactic :) That said, I'm happy to try, again. I may have been approaching this the wrong way and would be happy to take another stab at it.
